engine/render/renderer/renderer/DebugRenderer3D.cpp

```diff
@@ -47,16 +37,13 @@ void DebugRenderer3D::Initialise(const String& globalDataAttributeName,
 
 void DebugRenderer3D::Destroy()
 {
-    lineMaterial.DestroyMaterial();
+    lineMaterial.~Material();
```

Comment:
I don't think directly calling destructors is very nice

Reply:
As above.